test/DynamoCoreWpfTests/ViewExtensions/ViewExtensionTests.cs

```diff
@@ -87,11 +106,16 @@ public void Loaded(ViewLoadedParams p)
                 Counter++;
             };
 
-            var window = new Window
+            var window = new Window();
+            window.Closed += (sender, args) =>
             {
-                // Set the owner of the window to tuIDhe Dynamo window.
-                Owner = p.DynamoWindow
+                WindowClosed = true;
             };
+            if (SetOwner)
+            {
+                // Set the owner of the window to tuIDhe Dynamo window.
```

Comment:
Would you fix the word `tuIDhe`, I think you mean `the`?

Reply:
That's not mine, so I'll probably remove it :)